.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+
+node_js:
+  - 8
```

Comment:
iirc we went back to node 6 with the event hubs package. any reason not to do so here as well? if we support it, we should include it in CI

Reply:
was thinking of having min support to node.js 8.x since 10.x will become LTS this fall.

Reply:
Will be supporting 6.x onwards. Have updated the readme and target field in tsconfig.json as well.